influxdb3_server/src/query_executor.rs

```diff
@@ -468,6 +469,16 @@ impl<B: WriteBuffer> TableProvider for QueryTable<B> {
         TableType::Base
     }
 
+    fn supports_filters_pushdown(
+        &self,
+        filters: &[&Expr],
+    ) -> datafusion::common::Result<Vec<TableProviderFilterPushDown>> {
+        filters
+            .iter()
+            .map(|_f| Ok(TableProviderFilterPushDown::Inexact))
+            .collect()
```

Comment:
You're not using the exprs here at all nor does it need to be a Result since this will always be Ok. I think you can just do this:
```suggestion
        vec![TableProviderFilterPushDown::Inexact; filters.len()]
```

Like here:
https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=1d812cf42a2fb158006d0ab96ae346f3

Reply:
It'll have to be a result because that's the trait signature, but good call on the vec.